src/layer/image/index.ts

```diff
@@ -106,7 +108,9 @@ const SHADER_JSON_KEY = "shader";
 const SHADER_CONTROLS_JSON_KEY = "shaderControls";
 const CROSS_SECTION_RENDER_SCALE_JSON_KEY = "crossSectionRenderScale";
 const CHANNEL_DIMENSIONS_JSON_KEY = "channelDimensions";
-const VOLUME_RENDERING_JSON_KEY = "volumeRendering";
+// The volume rendering mode key is newer, but we still need to support the old one for backwards compatibility.
+const VOLUME_RENDERING_COMPAT_JSON_KEY = "volumeRendering";
```

Comment:
I think it would be simpler to just keep using the old key, and simply extend the allowed set of values.